tests/codemods/test_lazy_logging.py

```diff
@@ -0,0 +1,683 @@
+import pytest
+from tests.codemods.base_codemod_test import BaseSemgrepCodemodTest
+from core_codemods.lazy_logging import LazyLogging
+
+logging_funcs = {"debug", "info", "warning", "error", "critical"}
```

Comment:
Let's include `warn` here as well. I know we have a separate codemod to fix that but it means that the application of these codemods won't be sensitive to ordering.